### ISSUE TITLE
Vagrant shuts down the VM even with `KEEP_RUNNING=true`: is this expected?

### ISSUE THREAD

**Issue by brandonzylstra** [tmux-plugins/tmux-test#4]
The README says that `KEEP_RUNNING=true` results in _Vagrant_ staying up, but is that what was meant?   The real consumer of time is starting up the VM every time, so I was hoping that that what was meant.

Is it possible to keep the VM running between tests?

**Comment by bruno-** [comment]
Hi,
yes, the `KEEP_RUNNING` variable is used to keep VM running between the tests. Last time I ran the tests this was working.

One thing that wasn't emphasized in the readme: `KEEP_RUNNING=true` should go **at the beginning of the line** that runs the tests. So, the following code is a one liner: `KEEP_RUNNING=true ./run_tests`.

`KEEP_RUNNING=true` on a line by itself wont work.

**Comment by brandonzylstra** [comment]
Yes, putting the variable before the command is pretty standard, but does it keep the VM running between executions of the `run_tests` script, or just between tests if you run have multiple tests that the script kicks off?

If I run `KEEP_RUNNING=true ./run_tests` and I have only one test for it to run, should the VM stay up so that the next time I run `KEEP_RUNNING=true ./run_tests` it doesn't need to start up again?  Or should it only stay running between multiple tests that are all run the the same `KEEP_RUNNING=true ./run_tests` command?

**Comment by bruno-** [comment]
Yes, it should keep the VM running between the executions of the `run_tests` script.

I think [this](https://github.com/tmux-plugins/tmux-test/blob/master/run_tests#L95-L97) should clarify how things work. If `KEEP_RUNNING=true` is present, vagrant VM will NOT be halted after the `run_tests` script.

**Comment by brandonzylstra** [comment]
ok, I'll try to test again when I get home, but I'm quite sure the VM was going down after `run_tests`.  Are you seeing it stay up?
